docs/RELEASES.md

```
@@ -0,0 +1,74 @@
+- **N**: a counter that **resets at each stage boundary**.
+
+| stage | when | N counts |
+|---|---|---|
+| `alpha` | no stage tag in HEAD's ancestry | commits since the last GA tag |
+| `beta` | `v<target>-beta` is HEAD's most recent stage tag | commits since that tag, +1 (tagged commit is `.1`) |
+| `rc` | `v<target>-rc` is HEAD's most recent stage tag | commits since that tag, +1 |
+| GA | HEAD is at a `v<target>` tag | n/a — emit `<target>` verbatim |
+
+Example timeline starting from `v1.2.0`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)